python/dgl/graphbolt/data_block.py

```diff
@@ -30,17 +30,14 @@ class DataBlock:
     no node types, 'node_type' should be None.
     """
 
-    edge_feature: List[Dict[Tuple[str, str], torch.Tensor]] = None
+    edge_feature: Dict[Tuple[str, str], torch.Tensor] = None
     """Edge features associated with the 'sampled_subgraphs'.
     The keys are tuples in the format '(edge_type, feature_name)', and the
     values represent the corresponding features. In the case of a homogeneous
     graph where no edge types exist, 'edge_type' should be set to None.
-    Note 'edge_type' are of format 'str:str:str'.
     """
 
-    input_nodes: Union[
```

Comment:
For these changes, the related code also need to be changed. Same for all below fields.

Reply:
the changes I made is about annotation only. the running code lies in the use of data block such as `NegativeSampler`. I will update them in a following PR.

Reply:
Why it was Tuple[str, str, str]? Wasn't it node type?

Reply:
I think it was a typo